pkg/utils/docker.go

```diff
@@ -593,7 +593,7 @@ func DetermineDebugPortForPFE() (pfeDebugPort string) {
 
 // GetContainerTags of the Codewind version(s) currently running
 func GetContainerTags() ([]string, *DockerError) {
-	containerArr := baseImageNameArr
+	containerArr := containerImageNames
```

Comment:
Not directly related to this PR, especially since I named the array, but the `containerImageNames` array should probably just be called `containerNames`. (Containers and images being different things!)